client/app/components/contact-form.js

```diff
@@ -0,0 +1,43 @@
+import Ember from 'ember';
+import config from '../config/environment';
+const isTest = config.APP.STUB_EXTERNAL_MOCK;
+
+export default Ember.Component.extend({
+
+  districtContactEmail: '',
+  districtContactSubmitted: false,
+  districtContactErrored: false,
+
+  getSubmittedName: Ember.computed('name', function() {
+    return this.get('name') != undefined ? `${this.get('name')} ,` : '';
+  }),
+
+  actions: {
+    submitDistrictContact() {
+      this.set('districtContactErrored', false);
+
+      if (Ember.isPresent(this.get('districtContactEmail'))) {
+
+        Ember.$.ajax({
+          url: 'https://formspree.io/schoolbot-contact@vermonster.com',
+          method: 'POST',
+          dataType: 'json',
+          data: {
+            _replyto: this.get('districtContactEmail'),
+            _subject: `District contact by ${this.get('districtContactEmail')}`,
+            name: this.get('name'),
+            message: this.get('message')
```

Comment:
We should add these new properties with default values alongside `email` at the top – in addition to serving as documentation of what properties the component uses, I think this would make the `getSubmittedName` helper unnecessary because `name` would always be a valid string value.